### PR TITLE
Added posibility select option by text

### PR DESCRIPTION
**Pull request by spolischook**
According to conversations
minkphp/MinkSelenium2Driver#240
minkphp/Mink#710

### CODE REVIEW

**Comment by spolischook** [comment]
@stof please see this code. I don't know why but it works!)))

File: src/ZombieDriver.php [hunk 1]
```diff
@@ -506,32 +506,17 @@ public function setValue($xpath, $value)
...
-      for (i = 0; i < toUnselect.length; i++) {
-        toUnselect[i].selected = false;
-      }
-      for (i = 1; i < toSelect.length; i++) {
-        toUnselect[i].selected = true;
-      }
-      browser.selectOption(toSelect[0]);
-    }
+    value.forEach(function(optionValue) {
+        browser.select(node, optionValue);
```
Comment (aik099) [inline src/ZombieDriver.php]: This is not good, because it will trigger `change` JS event for each option being selected.

Comment (spolischook) [inline src/ZombieDriver.php]: If you do so by hand it doesn't?

Comment (aik099) [inline src/ZombieDriver.php]: In any case, there is a test that checks such behavior. If test passes, then all is good.

Comment (spolischook) [inline src/ZombieDriver.php]: Yep, all tests is green!

**Pull request closed without merging**